src/Tizen.NUI.Components/Attributes/ViewAttributes.cs

```diff
@@ -264,5 +280,89 @@ public override Attributes Clone()
             return new ViewAttributes(this);
         }
 
+        /// <summary>
+        /// Apply ViewAttributes to View.
+        /// </summary>
+        public virtual void ApplyToView(View view, ControlStates state = ControlStates.Normal)
+        {
+            if (view == null)
+            {
+                return;
+            }
+
+            if (Position != null)
+            {
+                view.Position = Position;
+            }
+            if (Size != null)
+            {
+                view.Size = Size;
+            }
+            if (MinimumSize != null)
+            {
+                view.MinimumSize = MinimumSize;
+            }
+            if (PositionUsesPivotPoint != null)
+            {
+                view.PositionUsesPivotPoint = PositionUsesPivotPoint.Value;
+            }
+            if (ParentOrigin != null)
+            {
+                view.ParentOrigin = ParentOrigin;
+            }
+            if (PivotPoint != null)
+            {
+                view.PivotPoint = PivotPoint;
+            }
+            if (WidthResizePolicy != null)
+            {
+                view.WidthResizePolicy = WidthResizePolicy.Value;
+            }
+            if (HeightResizePolicy != null)
+            {
+                view.HeightResizePolicy = HeightResizePolicy.Value;
+            }
+            if (SizeModeFactor != null)
+            {
+                view.SizeModeFactor = SizeModeFactor;
+            }
+
+            if (!appliedViews.ContainsKey(view))
+            {
+                appliedViews.Add(view, 0);
+                RegisterStateChangeCallback(view);
+            }
+        }
+
+        protected virtual void RegisterStateChangeCallback(View view)
+        {
+            view.FocusGained += (object sender, global::System.EventArgs e) =>
+            {
+                ApplyToView(view, ControlStates.Focused);
+            };
+
+            view.FocusLost += (object sender, global::System.EventArgs e) =>
+            {
+                ApplyToView(view, ControlStates.Normal);
+            };
+        }
+
+        public void DisApplyToView(View view)
```

Comment:
DisApply is not commoly used......   Please explain role of the function...

Reply:
User need to call DisApplyToView() before call the Dispose().

Reply:
If a View is removed then automatically Disposal will not run as there would be a handle to the View in this class, right?
Will the user know to call this API (DisApplyToView), seems like something that should be handled internally.

Reply:
This API is not actually removing the attributes from the View, it just removes the View from a container.  So the naming is not right.

Reply:
@rabbitfor 
"Does the button keep the Summer style or does it change its style to Winter?"
The style will change to "Winter"

@agnelovaz 
"Will the user know to call this API (DisApplyToView), seems like something that should be handled internally."

Because we don't want to let BaseCompoents.View depend the Tizen.NUI.Components, so we can't remove the view-handle in the View.Dispose().

Or we can add an event about dispose in the Basecomponents, so we don't need the method DisApplyToView. 

Do you think this is feasible?

Reply:
I would be worried about emitting a signal from a Dispose() function as I would expect the emitting object to have now been garbage collected.  So too late?  You could try it and share the result.